drape_frontend/text_layout.cpp

```diff
@@ -124,15 +124,15 @@ class TextOutlinedGeometryGenerator
 
 void SplitText(strings::UniString & visText, buffer_vector<size_t, 2> & delimIndexes)
 {
-  char const * delims = " \n\t";
-  size_t count = visText.size();
+  char constexpr delims[] = " \n\t";
+  size_t const count = visText.size();
   if (count > 15)
   {
     // split on two parts
     typedef strings::UniString::iterator TIter;
     auto const iMiddle = visText.begin() + count / 2;
 
-    size_t const delimsSize = strlen(delims);
+    size_t constexpr delimsSize = sizeof(delims)/sizeof(delims[0]);
```

Comment:
There is an error here anyway.
sizeof(delims) = 4, because you initialize from string literal.

Oh, if you decided to play here, make simple:
```
char constexpr delims[] = {' ', '\n', '\t'};
size_t constexpr delimsSize = std::size();
```

Reply:
Right, \0 is wrongly counted too. Char array is unreadable, subtracting -1 should be enough. Unfortunately, there is no constexpr version of strlen.